src/js/user-profile/containers/UserProfileApp.jsx

```diff
@@ -1,40 +1,47 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { removeInProgressForm } from '../../common/schemaform/save-load-actions';
 
 import UserDataSection from '../components/UserDataSection';
 import AuthApplicationSection from '../components/AuthApplicationSection';
 
 import RequiredLoginView from '../../common/components/RequiredLoginView';
 
 class UserProfileApp extends React.Component {
-  render() {
-    const view = (
-      <div className="row user-profile-row">
-        <div className="usa-width-two-thirds medium-8 small-12 columns">
-          <h1>Your Account</h1>
-          <div>
-            <AuthApplicationSection
-              userProfile={this.props.profile}
-              verifyUrl={this.props.verifyUrl}/>
-            <UserDataSection/>
-          </div>
-        </div>
-      </div>
-    );
-
-    return (
-      <div>
-        <RequiredLoginView
-          authRequired={1}
-          serviceRequired="user-profile"
-          userProfile={this.props.profile}
-          loginUrl={this.props.loginUrl}
-          verifyUrl={this.props.verifyUrl}>
-          {view}
-        </RequiredLoginView>
-      </div>
-    );
-  }
+
+ handleClick = (formId) => {
```

Comment:
I think it would be better to call this something more specific than `handleClick` here and in the props in subsequent components. It makes it easier to understand when looking at some of the other components.

Reply:
Great point, I will change here and should I also change this in src/js/messaging/components/buttons/ButtonDelete.jsx?